vllm/model_executor/layers/fused_moe/fused_moe.py

```diff
@@ -276,19 +292,20 @@ def invoke_fused_moe_kernel(A: torch.Tensor, B: torch.Tensor, C: torch.Tensor,
         B.stride(1),
         C.stride(1),
         C.stride(2),
-        B_scale.stride(0) if B_scale is not None and use_int8_w8a16 else 0,
-        B_scale.stride(1) if B_scale is not None and use_int8_w8a16 else 0,
+        B_scale.stride(0) if B_scale is not None and (use_int8_w8a16 or use_int8_w8a8) else 0,
+        B_scale.stride(1) if B_scale is not None and (use_int8_w8a16 or use_int8_w8a8) else 0,
         MUL_ROUTED_WEIGHT=mul_routed_weight,
         top_k=top_k,
         compute_type=compute_type,
         use_fp8_w8a8=use_fp8_w8a8,
+        use_int8_w8a8=use_int8_w8a8,
         use_int8_w8a16=use_int8_w8a16,
         **config,
     )
 
 
 def get_config_file_name(E: int, N: int, dtype: Optional[str]) -> str:
-    device_name = current_platform.get_device_name().replace(" ", "_")
+    device_name = torch.cuda.get_device_name().replace(" ", "_")
```

Comment:
What was the issue with using current_platform? We should really try to keep using that interface as we support other accelerators

Reply:
oh, sure, will revert it, it's just for some local testing purpose